tests/providerTests/inFormLayoutProviderTest.jsx

```diff
@@ -0,0 +1,38 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { FormLayoutProvider } from '../../src/lib/components/layout/FormLayout';
+
+export const inFormLayoutProviderTest = (Component) => {
+  it.each([
+    [
+      { layout: 'vertical' },
+      (rootElement) => {
+        expect(rootElement).toHaveClass('isRootInFormLayout');
+        expect(rootElement).toHaveClass('rootLayoutVertical');
+      },
+    ],
+    [
+      { layout: 'horizontal' },
+      (rootElement) => {
+        expect(rootElement).toHaveClass('isRootInFormLayout');
+        expect(rootElement).toHaveClass('rootLayoutHorizontal');
+      },
+    ],
+  ])('renders with FormLayout props: "%s"', (testedProps, assert) => {
+    const dom = render((
+      <FormLayoutProvider {...testedProps}>
+        {Component}
+      </FormLayoutProvider>
+    ));
+
+    assert(dom.container.firstChild);
+  });
+
+  it('renders without FormLayout', () => {
+    const dom = render((
```

Comment:
Again. Why multi line and multi brackets? This could be `render(Component)` and it is fine.

Reply:
Decided during a phone call with @bedrich-schindler that we do not want to deal with these things on PR. Either there should be an eslint for it or it is at the discretion of the developer.